src/database/queries/posts/getPostQuery.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-undef */
+const connection = require('../../config/connection');
+
+const getPostQuery = () => {
```

Comment:
`getPostsQuery` or `getAllPostsQuery` will be better than this name. Coz you retrieve all posts not single post you should add `s`